pkg/network/link/socket_listener.go

```diff
@@ -16,7 +16,7 @@
  * Copyright 2020 Red Hat, Inc.
  *
  */
-package dhcp
+package link
```

Comment:
Isn't this an UDP socket? If so, how is it fitting a link?
It is also dhcp specific, isn't it? Why move it out?

Reply:
Moving back to ./pkg/network/dhcp/server, I don't know why I did move it to link.